### PR TITLE
clean up logic for useFieldArray get default values

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/useFieldArray.ts [hunk 1]
```diff
@@ -104,10 +102,6 @@ export const useFieldArray = <
 
   allFields.current = fields;
 
-  if (isNameKey) {
-    set(fieldArrayDefaultValues.current, name, memoizedDefaultValues.current);
-  }
```
Comment (kotarella1110) [inline src/useFieldArray.ts:109]: cool!

Comment (kotarella1110) [inline src/useFieldArray.ts:109]: cool!

File: src/useFieldArray.ts [hunk 2]
```diff
@@ -239,7 +233,6 @@ export const useFieldArray = <
     const fieldValues = getCurrentFieldsValues();
     setFieldAndValidState(removeArrayAt(fieldValues, index));
     resetFields(removeArrayAt(fieldValues, index));
-    setIsDeleted(true);
```
Comment (kotarella1110) [inline src/useFieldArray.ts:242]: cool!

**Review by kotarella1110 — APPROVED** [review]
LGTM 💯

**Comment by bluebill1049** [comment]
Thank you @kotarella1110 for the review. 🙏

**Pull request merged**